sdk/eventhub/eventhubs-checkpointstore-blob/samples-dev/createCustomPipeline.ts

```diff
@@ -1,8 +1,8 @@
-/*
-  Copyright (c) Microsoft Corporation. All rights reserved.
-  Licensed under the MIT Licence.
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
 
-  This sample demonstrates how to create a custom `Pipeline` that the
+/**
+ * @summary Demonstrates how to create a custom `Pipeline` that the
```

Comment:
This might be regrettable, but it's important that JSDoc block comments start each line, even blank lines, with a space and an asterisk. So,

```javascript
/**
 * @summary <summary here>
 * 
 * More text can follow.
 */

Reply:
Oh sure, that makes sense. Normally my editor would take care of that for me but I think since I copy pasted I missed that.

Reply:
@witemple-msft  - worth encoding this into the linter rules?